lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -3226,8 +3227,9 @@ public function mapEmbedded(array $mapping)
      *
      * @param string $property
      * @param ClassMetadataInfo $embeddable
+     * @param boolean $nullable
      */
-    public function inlineEmbeddable($property, ClassMetadataInfo $embeddable)
+    public function inlineEmbeddable($property, ClassMetadataInfo $embeddable, $nullable)
```

Comment:
Is that extra parameters really required? Can't you use `$this->embeddedClasses[$property]['nullable']` in this method instead?